src/writer/to_code.rs

```diff
@@ -116,12 +116,21 @@ impl ToCode for Chunk {
                 let s = format_block_one_line(&prefix, suffix, &value_strings, "", "");
                 vec![s]
             }
-            ErrorResultReturn { ref value } => {
+            AssertErrorSanity => {
+                let assert = "assert_eq!(is_ok == 0, !error.is_null());";
```

Comment:
Why not use the ffi value the gboolean here? It would be more readable that way

Reply:
I'm not sure I'm correctly understanding the suggestion.
Isn't gboolean a type alias to an int, in Rust?
Or are you focusing on the `0` and want to replace that with `glib_sys::GFALSE`?